apps/meteor/app/2fa/client/TOTPOAuth.js

```diff
@@ -105,7 +105,7 @@ Meteor.loginWithLinkedin = function (options, cb) {
 	overrideLoginMethod(loginWithLinkedin, [options], cb, loginWithLinkedinAndTOTP);
 };
 
-Accounts.onPageLoadLogin((loginAttempt) => {
+Accounts.onPageLoadLogin(async (loginAttempt) => {
```

Comment:
Should this be a promise.await for now? Accounts.onPageLoadLogin doesn't seems to accept an async callback, so 2fa may not work 👀

Reply:
you are right, but since process2faReturn is already async function I dont believe running this sync is the required